src/editors/containers/ProblemEditor/index.test.tsx

```diff
@@ -1,127 +1,125 @@
-import 'CourseAuthoring/editors/setupEditorTest';
 import React from 'react';
-import { shallow } from '@edx/react-unit-test-utils';
-import { Spinner } from '@openedx/paragon';
-import { thunkActions, selectors } from '../../data/redux';
-import { RequestKeys } from '../../data/constants/requests';
-import { ProblemEditorInternal as ProblemEditor, mapStateToProps, mapDispatchToProps } from '.';
+import { render, screen, initializeMocks } from '../../../testUtils';
+import { ProblemEditorInternal } from './index';
+import messages from './messages';
 
-jest.mock('./components/EditProblemView', () => 'EditProblemView');
-jest.mock('./components/SelectTypeModal', () => 'SelectTypeModal');
-
-jest.mock('react', () => {
-  const updateState = jest.fn();
-  return {
-    ...jest.requireActual('react'),
-    updateState,
-    useState: jest.fn(val => ([{ state: val }, jest.fn().mockName('setState')])),
-  };
+// Mock child components for easy selection
+jest.mock('./components/SelectTypeModal', () => function mockSelectTypeModal(props: any) {
+  return <div>SelectTypeModal {props.onClose && 'withOnClose'}</div>;
+});
+jest.mock('./components/EditProblemView', () => function mockEditProblemView(props: any) {
+  return <div>EditProblemView {props.onClose && 'withOnClose'} {props.returnFunction && 'withReturnFunction'}</div>;
 });
 
-jest.mock('../../data/redux', () => ({
-  thunkActions: {
-    problem: {
-      initializeProblemEditor: jest.fn().mockName('thunkActions.problem.initializeProblem'),
-    },
-  },
-  selectors: {
-    app: {
-      blockValue: jest.fn(state => ({ blockValue: state })),
-      shouldCreateBlock: jest.fn(state => ({ shouldCreateBlock: state })),
-    },
-    problem: {
-      problemType: jest.fn(state => ({ problemType: state })),
-    },
-    requests: {
-      isFinished: jest.fn((state, params) => ({ isFinished: { state, params } })),
-      isFailed: jest.fn((state, params) => ({ isFailed: { state, params } })),
-    },
-  },
-}));
+const baseProps = {
+  onClose: jest.fn(),
+  returnFunction: jest.fn(),
+  initializeProblemEditor: jest.fn(),
+  blockValue: { foo: 'bar' },
+};
+
+describe('ProblemEditorInternal', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    initializeMocks();
```

Comment:
```suggestion
    initializeMocks();
```
`initializeMocks()` already calls `jest.clearAllMocks()` for you, so you don't need both.

Reply:
Done.